sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md

```diff
@@ -1,7 +1,23 @@
 # Change Log azure-data-appconfiguration
+## Version 1.0.0-preview.6 (2019-10-25)
+For details on the Azure SDK for Java (October 2019 Preview) release refer to the [release announcement](https://aka.ms/azure-sdk-preview6-java).
+- renaming addSetting, getSetting, deleteSetting, setSetting, listSettings, listSettingRevisions to
```

Comment:
renaming -> Renamed
The tense of the other line items in previous release are past-tense. I'd unify these verbs.